src/message_interface/mod.rs

```diff
@@ -107,12 +107,6 @@ where
     handle.listen(events, handler).await;
 }
 
-#[cfg(feature = "events")]
-/// Remove wallet event listeners, empty vec will remove all listeners
-pub async fn clear_listeners(handle: &WalletMessageHandler, events: Vec<WalletEventType>) {
```

Comment:
We can't remove this, it's being used by the other bindings

Reply:
Actually no, i moved it to directly calling the account_manager.clear_listeners()